Add Descriptions: testing, tutorials & upgrading (#2833)
vaadin/docs: articles/tutorial/components-and-layouts.adoc

title: Create a View
order: 40
page-title: Vaadin Flow Basics. Build a View With Components and Layouts
description: Creating views with components and layouts.
---

= Create a Vaadin Flow View with Components

> Vaadin Flow uses an event-driven component model for building UIs. It's a Java framework for building web applications. It has a component-based programming model that allows you to build user interfaces.

Reviewer: ```suggestion
Vaadin Flow is a Java framework for building web applications. It has a event-driven component-based programming model that allows you to build user interfaces.
```
